script.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+//8
+document.body.addEventListener('click', (event) => {
+  event.preventDefault();
+  const x = event.pageX;
```

Comment:
when I use this solution, I have some problem, but when I change `pageX` to `clientX` the problem disappears